src/frontend/src/hooks/UseInstance.tsx

```diff
@@ -8,7 +8,8 @@ import { type PathParams, apiUrl } from '../states/ApiState';
 export interface UseInstanceResult {
   instance: any;
   setInstance: (instance: any) => void;
-  refreshInstance: () => Promise<QueryObserverResult<any, any>>;
+  refreshInstance: () => void;
+  refreshInstancePromise: () => Promise<QueryObserverResult<any, any>>;
```

Comment:
What is the reason for this change? You don't need to await the Promise if you don't want to where you use `refreshInstance`?

Reply:
Dosen something like:
```diff
 return {
    sessionData,
    setSessionData,
    sessionId,
-    refreshSession
+    refreshSession: () => { refreshSession() },
    sessionQuery,
    status,
```

would fix the error to, but without cluttering the useInstance hook with unnecessary functions.

Reply:
I do not see how this would fix the issue

Reply:
The additional wrapper promise function does seem clunky.. I'm sure there should be a better way to suppress the warning

Reply:
I do note really care how this is solved but there are 28 issues with severity middle caused by the underlying change - those should be addresses https://sonarcloud.io/project/issues?rules=typescript%3AS6544&issueStatuses=OPEN%2CCONFIRMED&id=inventree_InvenTree